hangul/hangulkeyboard.c

```diff
@@ -815,6 +823,53 @@ hangul_keyboard_list_load_dir(const char* path)
 
     globfree(&result);
     free(pattern);
+#elif _WIN32
+    WIN32_FIND_DATA FindFileData;
+    HANDLE hFind;
+    int n = strlen(pattern) * sizeof(TCHAR) + 1;
+
+    TCHAR *wpattern = (LPTSTR*)malloc(n);
+    if (wpattern == NULL) {
+	free(pattern);
+	return 0;
+    }
+
+    MultiByteToWideChar(CP_ACP, 0, pattern, strlen(pattern), wpattern, n);
+
+    hFind = FindFirstFile(wpattern, &FindFileData);
```

Comment:
그런데 WideCharToMultiByte함수를 사용할 것이므로
```suggestion
    hFind = FindFirstFileW(wpattern, &findFileData);
```
타입도 WCHAR를 사용하고, 명시적으로 W함수를 부르는 것이 좋을 것 같은데요, 의견 부탁드립니다.

Reply:
CMake / msbuild의 경우 `UNICODE`, `_UNICODE`가 기본 정의되어 컴파일되므로 `FindFirstFileW`로 명시하는게 낫겠네요. `TCHAR`도 일단은 `WCHAR`로 모두 고치겠습니다.